src/discord.coffee

```diff
@@ -39,9 +56,29 @@ class DiscordBot extends Adapter
         @client.on 'ready', @.ready
         @client.on 'message', @.message
         @client.on 'disconnected', @.disconnected
+        @client.on 'messageReactionAdd', (message, user)  => 
+          @.message_reaction('reaction_added', message, user)
```

Comment:
Why the underscores all of a sudden?

Reply:
I'm pretty sure I used underscores to begin with and someone changed things to camel case along the way....

Reply:
but underscores aren't standard in javascript or cs unless they're constants, which are all caps with underscores (like java)